### PR TITLE
Polyline demo : names for items created on the fly

### PR DESCRIPTION
**Pull request by maxGimeno**
This PR adds the possibility to choose the name of the point_sets and polylines created on the fly, but keeps the same counter.

### CODE REVIEW

**Comment by sloriot** [comment]
I think it is confusing that the background of the text box is gray..
For the naming, I would use the following stategy:

- Make the text box empty by default (and clear it after each polyline creation)
- If the item's name was empty use "Polyline #X" as name, and the value of the text otherwise (no "#X")

@maxGimeno what do you think?

**Comment by maxGimeno** [comment]
The background of the text box is supposed to be white, I don't see it gray. This is strange

**Comment by sloriot** [comment]
screenshot from 2016-04-22 10-39-43
I only compiled the polyline_io_plugin

**Comment by sloriot** [comment]
Thanks for the naming, working great.

**Comment by maxGimeno** [comment]
What It looks like this on Fedora :
screen
I guess it is the default style Qt uses on your OS that makes it so even.

**Comment by maxGimeno** [comment]
I tried darkening the background, can you test and tell me if it's better ?

File: Polyhedron/demo/Polyhedron/Plugins/IO/Add_point_set_dialog.ui [hunk 1]
```diff
@@ -10,6 +10,421 @@
...
+        <red>0</red>
+        <green>0</green>
+        <blue>0</blue>
+       </color>
+      </brush>
+     </colorrole>
+    </disabled>
+   </palette>
+  </property>
   <property name="windowTitle">
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Plugins/IO/Add_point_set_dialog.ui]: I strongly disagree with that sort of modification of the default look and feel. We are not artists, and we better use the default styles on different platforms!

Comment (maxGimeno) [inline Polyhedron/demo/Polyhedron/Plugins/IO/Add_point_set_dialog.ui]: Yes, but apparently the default style on @sloriot 's OS makes it confusing. It's just slightly darker, to reinforce the contrast. If it doesn't work I guess we'll just keep the default confusing one.

Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Plugins/IO/Add_point_set_dialog.ui]: Any modification of the style we make will appear even more confusing to more users. If Sébastien does not like the look and feel of Qt5 on his machine, he can try another style.

We, at GeometryFactory, cannot write good styles. That is a job, a difficult one!

**Comment by sloriot** [comment]
I fixed the line-edit background issue. PR ready now.

**Comment by maxGimeno** [comment]
No it's not
I have another branch in it, Mesh_3 shouldn't be affected by this PR

**Comment by sloriot** [comment]
Fixed

**Comment by sloriot** [comment]
Also tested locally

**Pull request merged**